salt/states/pkg.py

```diff
@@ -2678,6 +2678,18 @@ def mod_aggregate(low, chunks, running):
                     elif 'name' in chunk:
                         version = chunk.pop('version', None)
                         if version is not None:
+                            if version == 'latest':
+                                try:
+                                    version = __salt__['pkg.latest_version'](chunk['name'],
+                                                                             fromrepo=chunk.get('fromrepo'),
+                                                                             refresh=chunk.get('refresh'))
```

Comment:
yeah, so that could end up with some weird behavior in aggregate, but it should be the same weird behavior as in pkg.installed